liquidity/parameters.go

```diff
@@ -528,6 +551,16 @@ func ParametersToRpc(cfg Parameters) (*clientrpc.LiquidityParameters,
 		addrType = clientrpc.AddressType_ADDRESS_TYPE_UNKNOWN
 	}
 
+	easyAssetMap := make(map[string]*clientrpc.EasyAssetParams, len(
+		cfg.AssetAutoloopParams,
```

Comment:
the problem here is, if len(cfg.AssetAutoloopParams) here is 0 then the easyAssetMap will only have a nil entry which will crash in the client. So we should add at least one entry here.